java/src/jmri/jmrix/can/cbus/CbusLightManager.java

```diff
@@ -87,14 +89,58 @@ public boolean allowMultipleAdditions(String systemName) {
         return true;
     }
 
+    /* createSystemName() and getNextValidAddress() are not used in code, or any other LightManager
```

Comment:
@icklesteve perhaps you had a reason preparing for new features to add these methods, but no other LightManagers have them, and no code currently calls them either. So I commented them out for now, as well as the tests. OK to remove?

Reply:
Absolutely @silverailscolo , no point in maintaining dead code!
Thanks for spotting :-)